action/action_test.go

```diff
@@ -40,7 +40,7 @@ var mockSet = credentials.Set{
 func newClaim() *claim.Claim {
 	now := time.Now()
 	return &claim.Claim{
-		SchemaVersion: claim.DefaultSchemaVersion,
+		SchemaVersion: "1.0.0-DRAFT+d7ffba8",
```

Comment:
Apologies if this has been asked already and I am just missing it but is there a reason why you aren't using `claim.GetDefaultSchemaVersion()` here instead of copying the string value in a few places throughout the tests?

Reply:
Not sure my train of thought; nevertheless, updated to use that method instead of hard-coding.